### PR TITLE
fastsim 2017 PU and pmx wfs

### PR DESCRIPTION
**Pull request by nwickram**
added workflows for fastsim 2017 PU and premix samples

### CODE REVIEW

**Comment by zhenhu** [comment]
Hi @angirar and @ssekmen , this pull request is related to PR #23363 , where you only have the noPU relval workflows. 
In this PR, we further added PU and premix workflows. 
Could you please double check if all the conditions and eras are consistent with your expectation?

**Comment by zhenhu** [comment]
assign fastsim

File: Configuration/PyReleaseValidation/python/relval_premix.py [hunk 1]
```diff
@@ -80,3 +80,18 @@
...
 workflows[250409] = ['SMS-T1tttt_mGl-1500_mLSP-100_13',["FS_SMS-T1tttt_mGl-1500_mLSP-100_13_PRMXUP15_PU25","HARVESTUP15FS","MINIAODMCUP15FS"]]
+
+#fastsim,  2017 PU50####
+
+## premixed minbias
+workflows[250399.17]=['',['FS_PREMIXUP17_PU50']]
+## signal + PU
+workflows[250400.17] = ['ZEE_13',["FS_ZEE_13_PRMXUP17_PU50","HARVESTUP17FS","MINIAODMCUP17FS"]]
+workflows[250402.17] = ['TTbar_13',["FS_TTbar_13_PRMXUP17_PU50","HARVESTUP17FS","MINIAODMCUP17FS"]]
+workflows[250402.117] = ['TTbar_13',["FS_PREMIXUP17_PU50","FS_TTbar_13_PRMXLOCALUP17_PU50","HARVESTUP17FS"]]
```
Comment (makortel) [inline Configuration/PyReleaseValidation/python/relval_premix.py]: Could you please use `.171` instead to follow the convention of similar FullSim workflows?

Comment (zhenhu) [reply Configuration/PyReleaseValidation/python/relval_premix.py]: good point. Thanks for your comment.

**Comment by ssekmen** [comment]
@nwickram , @zhenhu , thank you very much.  This looks good to me.  @angirar , please confirm.

**Comment by zhenhu** [comment]
please test workflow 25400.17, 25402.17, 250399.17, 250400.17, 250402.17, 2017.8

**Comment by angirar** [comment]
@zhenhu @nwickram @ssekmen Thank you all. The changes look fine and consistent to me. Although I do want to ask about why only 50ns PU workflows are being added for 2017 FastSim? Since standard workflows for both 2017 & 2018 have 25ns PU. Forgive me if I'm not aware of this discussion.

**Comment by kpedro88** [comment]
@angirar "PU50" is mean number of interactions per bunch crossing = 50, not 50ns. The "25" at the start of the WF name implies that it's 25ns.

**Comment by angirar** [comment]
@kpedro88 Thanks for the clarification.

**Comment by zhenhu** [comment]
please test workflow 25400.17, 25402.17, 250402.171, 250399.17, 250400.17, 250402.17, 2017.8

**Comment by davidlange6** [comment]
Your PR is unmergeable. Please have a look and possibly rebase it.

**Comment by fabiocos** [comment]
@nwickram the PR needs to be rebased

**Comment by zhenhu** [comment]
please test workflow 25400.17, 25402.17, 250402.171, 250399.17, 250400.17, 250402.17, 2017.8

**Comment by zhenhu** [comment]
please abort

**Comment by zhenhu** [comment]
please test workflow 25400.17, 25402.17, 250402.171, 250399.17, 250400.17, 250402.17, 2017.8

**Comment by zhenhu** [comment]
+1

**Comment by ssekmen** [comment]
+1

**Comment by kpedro88** [comment]
+upgrade

**Comment by fabiocos** [comment]
+1

**Pull request merged**